src/main/java/com/auction/auction/bid/controller/AuctionController.java

```diff
@@ -63,29 +49,16 @@ public List<Auction> getAllAuctions() {
         return auctionService.getAllAuctions();
     }
 
-    @GetMapping(value="auction/{title}")
-    public List<Auction> getAuctionsByTitle(@PathVariable String title) {
-        return auctionService.getAuctionsByTitle(title);
-    }
 
-    @GetMapping(value="auction/{seller}")
-    public List<Auction> getAuctionsBySeller(@PathVariable String seller) {
-        return auctionService.getAuctionsBySeller(seller);
+    @GetMapping(value="auction/{owner}")
```

Comment:
i thing here you are using an ID of the owner but not some Owner object, so it must be named as ownerId